docs/lib/analytics/fragments/android/getting-started/30_initAnalytics.md

```diff
@@ -0,0 +1,25 @@
+Add the following imports to the top of your `MainActivity.java` file:
+
+```java
+com.amplifyframework.analytics.pinpoint.AmazonPinpointAnalyticsPlugin;
+com.amplifyframework.auth.Cognito.AmazonCognitoAuthPlugin;
+com.amplifyframework.core.Amplify;
+com.amplifyframework.analytics.AnalyticsEvent;
+```
+
+Add the following code to the onCreate() method of `MainActivity.java`
+
+```java
+try {
+        AmazonCognitoAuthPlugin authPlugin = new AmazonCognitoAuthPlugin((Application) context)
+        AmazonPinpointAnalyticsPlugin analyticsPlugin = new AmazonPinpointAnalyticsPlugin((Application) context);
+        Amplify.addPlugin(authPlugin)
+        Amplify.addPlugin(analyticsPlugin);
+        Amplify.configure(context);
+    } catch (Exception e) {
+        Log.e("GetStarted", "Error initializing", e);
+    }
+
+    Analytics.recordEvent("GetStarted");
```

Comment:
Let's rename this to something else that isn't the tag we're using above. How about `AppOpened`?